Makefile

```diff
@@ -115,16 +115,26 @@ tidy: #HELP Update dependencies.
 	# Force tidy to use the version already in go.mod
 	$(Q)go mod tidy -go=$(GOLANG_VERSION)
 
+.PHONY: catalogd-manifests #EXHELP Generate catalogd code and catalogd manifests.
+KUSTOMIZE_CRDS_DIR := catalogd/config/base/crd/bases
+KUSTOMIZE_RBAC_DIR := catalogd/config/base/rbac
+KUSTOMIZE_WEBHOOKS_DIR := catalogd/config/base/manager/webhook
+catalogd-manifests: $(CONTROLLER_GEN) 
```

Comment:
I would suggest sticking with just `manifests` and adding catalogd's controller-gen commands to that existing recipe instead of splitting out separate component-based manifests targets. As we consolidate further, and start looking toward kustomize consolidation, we'll be able to refactor some of the controller-gen commands to generate catalogd and operator-controller manifests at the same time (e.g. the CRDs)

Reply:
Either way is fine with me. Will fix it

Reply:
Fixed in commit https://github.com/operator-framework/operator-controller/commit/d06df7ff15ac685729983f5fc8890a923da1aeea